.travis.yml

```diff
@@ -37,8 +37,10 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/gRPC TOOLCHAIN=clang-cxx17 VERBOSE=0
 
-    - os: linux
-      env: PROJECT_DIR=examples/gRPC TOOLCHAIN=gcc-7-cxx17
+    # FIXME: https://travis-ci.com/rbsheth/hunter/jobs/132120867
+    # Somtimes results in an internal compiler error
```

Comment:
Can you try to set environment variable `HUNTER_JOBS_NUMBER` to `1`?
* https://docs.hunter.sh/en/latest/reference/user-variables.html#id14

Reply:
Tried, still failed: https://travis-ci.org/ingenue/hunter/jobs/405158798